docker/fedora/Dockerfile

```diff
@@ -21,6 +21,6 @@ RUN yum install -y \
 ENV PATH "/root/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
 
```

Comment:
Is there some way we can read in the `rust-toolchain` so we do not have to constantly update this?

Reply:
Or do we go to a Dockerfile.in setup?

Reply:
I'm not at all familiar with Docker but the way it's set up right now is that we have a base `Dockerfile.in` that is used to build the other two. It reads in the toolchain version when the build script is run

Reply:
got it.